src/navigation/navigator.js

```diff
@@ -10,21 +11,29 @@ import RegistrationScreen from '../screens/RegistrationScreen'
 import ProfileScreen from '../screens/ProfileScreen'
 import CompleteScreen from '../screens/CompleteScreen'
 
+import { useTranslation } from 'react-i18next'
+
 /**
  * WizardNavigator navigates between screens.
  * @type {NavigationNavigator}
  * @component
  */
-const WizardNavigator = createStackNavigator({
-  SplashScreen: SplashScreen,
-  TandC: TermsAndConditionsScreen,
-  Registration: RegistrationScreen,
-  Home: HomeScreen,
-  Map: MapScreen,
```

Comment:
Between MaScreen and UnitScreen we need `DimensionScreen`.

Show: when a stage on the map has been selected
Cancel: back to map
Selected dimension: go to `UnitScreen`